app/Models/Product.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Product extends Model
+{
+    protected $fillable = [
+        'name',
+        'category_id',
+        'suplier_id',
+        'price',
+        'price_discount',
+        'description',
+        'weight',
+    ];
+
+    public function supplier()
+    {
+        return $this->belongsTo('App\Models\Supplier', 'suplier_id');
```

Comment:
```suggestion
        return $this->belongsTo(Supplier::class, 'suplier_id');
```
sửa hết các phần tương tự

Reply:
e sửa rồi ạ